langstream-runtime/langstream-runtime-impl/src/main/python/langstream_runtime/tests/test_runtime.py

```diff
@@ -24,25 +25,97 @@
 from langstream_runtime import runtime
 
 
-def test_simple_agent():
+@patch('time.time_ns', return_value=12345000000)
+def test_simple_agent(_):
     config_yaml = f"""
         streamingCluster:
             type: kafka
         agent:
             applicationId: testApplicationId
             agentId: testAgentId
+            agentType: testAgentType
             configuration:
                 className: langstream_runtime.tests.test_runtime.TestAgent
     """
     config = yaml.safe_load(config_yaml)
     agent = runtime.init_agent(config)
-    runtime.run(config, agent=agent, max_loops=2)
-    assert agent.context['config'] == [{'className': 'langstream_runtime.tests.test_runtime.TestAgent'}]
-    assert agent.context['start'] == 1
-    assert agent.context['close'] == 1
-    assert agent.context['set_commit_callback'] == 1
-    assert agent.context['records'][0].value() == 'some record 0 processed'
-    assert agent.context['records'][1].value() == 'some record 1 processed'
+    agent_info = runtime.AgentInfo()
+    runtime.run(config, agent=agent, agent_info=agent_info, max_loops=2)
+    print(json.dumps(agent_info.worker_status(), indent=2))
+    assert json.dumps(agent_info.worker_status(), indent=2) == """[
+  {
+    "agent-id": "testApplicationId-testAgentId",
```

Comment:
In Java the agent-id name here is testAgentId.
Should we change in Java or Python ?

Reply:
please change it in Python, the "agentId" should be unique in the scope of the application, not globally unique in the tenant

side topic: currently we have a problem if you explicitly set the same agentId in two pipelines of the same application, but the problem is to be solved in the planner

Reply:
done